docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/30-common-configurations.md

```diff
@@ -0,0 +1,423 @@
+
+:::note
+When making use of the self-signed certificate option, cert-manager must be
+installed in the cluster.
+:::
+
+To supply your own certificate, set the following configuration:
+
+```yaml
+api:
+  tls:
+    selfSignedCert: false
+```
+
+When setting `selfSignedCert` to `false`, a
+[TLS `Secret`](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets)
+with the name `kargo-api-cert` is expected to exist in the same namespace as
+the API server.
+
+#### Terminating TLS
+
+In certain cases, you may want to disable TLS on the API server because it is
+terminated at the [`Ingress`](#api-ingress) level. To do this, set the following
+configuration:
+
+```yaml
+api:
+  tls:
+    enabled: false
+    terminatedUpstream: true
+```
+
+:::info
+When setting `terminatedUpstream` to `true`, the API server will continue to
+generate URLs with the `https` scheme, but will not enforce TLS.
+:::
+
+### API Ingress
+
+:::info
+Instead of making use of an `Ingress` resource, you can also expose the API
+server using a `LoadBalancer` or `NodePort` service type. Refer to the
+[API Service](#api-service) section for more information.
+:::
+
+By default, Kargo will not create an `Ingress` resource for the API server
+and will only be accessible within the cluster or through the API server's
+[`Service` resource](#api-service). To expose the API server to the internet,
+an `Ingress` resource can be created.
+
+:::caution
+Enabling the API server Ingress without proper configuration can expose the API
+server to the internet in an insecure manner. Refer to the
+[Secure Configuration](../40-security/10-secure-configuration.md) section for
+more information on securing the API server.
+:::
+
+To configure the API server to use an `Ingress` resource, set the following
+configuration:
+
+```yaml
+api:
+  ingress:
+    enabled: true
+    ingressClassName: nginx
+```
+
+#### Ingress TLS
+
+By default, Kargo will enable TLS on the `Ingress` resource using a self-signed
+certificate using [cert-manager](https://cert-manager.io). 
+
+:::note
+When making use of the self-signed certificate option, cert-manager must be
+installed in the cluster.
+:::
+
+To supply your own certificate, set the following configuration:
+
+```yaml
+api:
+  ingress:
+    tls:
+      selfSignedCert: false
+```
+
+When setting `selfSignedCert` to `false`, the `Ingress` resource expects a
+[TLS `Secret`](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)
+with the name `kargo-api-ingress-cert` to exist in the same namespace as the
+API server.
+
+## Git Configuration
+
+Kargo supports a number of Git-related configurations that can be set at
+installation time.
+
+### Default Commit Author
+
+To set the default commit author Kargo will use when committing changes using
+the [`git-commit` Promotion step](../../50-user-guide/60-reference-docs/30-promotion-steps/git-commit.md),
+the following configuration can be set (shown here with the default values):
```

Comment:
Is it worth mentioning that users can override this default in their own configurations? Or leave that the the step docs you're already linking to? I could go either way.

Reply:
The intention of the link was to point people towards the overwrite.

Reply:
Fair enough. I suppose the word "default" also implies is well enough that it's overridable, even if the reader doesn't follow the link to the git-commit step.